cluster-up/kubectl.sh

```diff
@@ -30,7 +30,7 @@ source ${KUBEVIRTCI_PATH}/hack/common.sh
 source ${KUBEVIRTCI_CLUSTER_PATH}/$KUBEVIRT_PROVIDER/provider.sh
 source ${KUBEVIRTCI_PATH}/hack/config.sh
 
-if [ "$1" == "console" ] || [ "$1" == "vnc" ]; then
+if [ "$1" == "console" ] || [ "$1" == "vnc" ] || [ "$1" == "migrate" ] || [ "$1" == "start" ] || [ "$1" == "stop" ]; then
```

Comment:
kubectl.sh is vendored from kubevirt-ci.  you'll have to make the change in kubevirt-ci then update the `kubevirtci_git_hash` in hack/config-default.sh

Reply:
Ah thanks, @davidvossel completely forgot about this.